integration/spring-boot/autoconfigure/src/main/java/org/apache/cxf/spring/boot/autoconfigure/MetricsProperties.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.cxf.spring.boot.autoconfigure;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("management.metrics")
```

Comment:
We should probably better use `cxf.management.metrics` (or just `cxf.metrics`) in order to not conflict / confuse with Spring Boot Actuator bundled properties.

Reply:
I see your pont, but as I see, all of Spring Boot related metrics are under management.metrics too. 
https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html#production-ready-metrics-meter
As a developer, I'll search it here. Otherwise, someone should update the documentation too (I don't have a write access to this one).

Reply:
Fair enough, it is better to follow the Spring's recommendation, https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html#per-meter-properties, so fe we could have metrics under `management.metrics.cxf` and able to control/configure all or part of them, does it make sense?

Reply:
`management.metrics.cxf` is OK for me.

Reply:
There are some [guidelines about this](https://docs.spring.io/spring-boot/docs/2.2.5.RELEASE/reference/htmlsingle/#boot-features-custom-starter-configuration-keys) in Spring Boot's documentation. As per those guidelines, please don't use `management.*` for third-party properties. +1 for properties that begin `cfx.`.

Reply:
Hi @wilkinsona I overlooked this important thing, thanks. I'll moving that.